src/components/Header.vue

```diff
@@ -55,9 +46,12 @@
     },
     data() {
       return {
-        bcgovLogo: `/images/bcgov-logo.svg`,
-        bcgovLogoText: `/images/bcgov-logo-text.svg`,
-        bcgovLogoPrint: `/images/bcgov-logo-print.png`,
+        // bcgovLogo: `/images/bcgov-logo.svg`,
+        // bcgovLogoText: `/images/bcgov-logo-text.svg`,
+        // bcgovLogoPrint: `/images/bcgov-logo-print.png`,
```

Comment:
Can we still use these SVGs. They are the same as the ones you added, just renamed. Instead of `bcidSymbol`, `bcidLogoRev`, and `bcidMainLogoRev`.

Reply:
Will do. I'll use the older svg and delete the new svgs.